modules/stochastic_tools/test/include/utils/TestDistributionDirectPostprocessor.h

```diff
@@ -0,0 +1,51 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef TESTDISTRIBUTIONDIRECTPOSTPROCESSOR_H
+#define TESTDISTRIBUTIONDIRECTPOSTPROCESSOR_H
+
+#include "GeneralPostprocessor.h"
+
+class TestDistributionDirectPostprocessor;
```

Comment:
suggest -> ``TestNormalDistributionDirectPostprocessor``

Reply:
In the future we might want to add tests for interfaces for different distributions, this class will work for that.

Reply:
👍